src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "altinn-app-frontend",
-  "version": "3.13.6",
+  "version": "3.14.6",
```

Comment:
```suggestion
  "version": "3.14.0",
```

Could potentially need to change if other pr's are merged.

Reply:
Thats fine, I'm now used that, hehe